lib/mrsk/cli/main.rb

```diff
@@ -76,49 +76,54 @@ def redeploy
+    run_hook "post-deploy", runtime: runtime.round
   end
 
   desc "rollback [VERSION]", "Rollback app to VERSION"
   def rollback(version)
-    with_lock do
-      invoke_options = deploy_options
+    rolled_back = false
+    runtime = print_runtime do
+      with_lock do
+        invoke_options = deploy_options
 
-      hold_lock_on_error do
-        MRSK.config.version = version
-        old_version = nil
+        hold_lock_on_error do
+          MRSK.config.version = version
+          old_version = nil
 
-        if container_available?(version)
-          say "Start version #{version}, then wait #{MRSK.config.readiness_delay}s for app to boot before stopping the old version...", :magenta
+          if container_available?(version)
+            say "Start version #{version}, then wait #{MRSK.config.readiness_delay}s for app to boot before stopping the old version...", :magenta
 
-          on(MRSK.hosts) do
-            execute *MRSK.auditor.record("Tagging #{MRSK.config.absolute_image} as the latest image"), verbosity: :debug
-            execute *MRSK.app.tag_current_as_latest
-          end
+            on(MRSK.hosts) do
+              execute *MRSK.auditor.record("Tagging #{MRSK.config.absolute_image} as the latest image"), verbosity: :debug
+              execute *MRSK.app.tag_current_as_latest
+            end
 
-          on(MRSK.hosts) do |host|
-            roles = MRSK.roles_on(host)
+            on(MRSK.hosts) do |host|
+              roles = MRSK.roles_on(host)
 
-            roles.each do |role|
-              app = MRSK.app(role: role)
-              old_version = capture_with_info(*app.current_running_version).strip.presence
+              roles.each do |role|
+                app = MRSK.app(role: role)
+                old_version = capture_with_info(*app.current_running_version).strip.presence
 
-              execute *app.start
+                execute *app.start
 
-              if old_version
-                sleep MRSK.config.readiness_delay
+                if old_version
+                  sleep MRSK.config.readiness_delay
 
-                execute *app.stop(version: old_version), raise_on_non_zero_exit: false
+                  execute *app.stop(version: old_version), raise_on_non_zero_exit: false
+                end
               end
             end
-          end
 
-          audit_broadcast "Rolled back #{service_version(Mrsk::Utils.abbreviate_version(old_version))} to #{service_version}" unless options[:skip_broadcast]
-        else
-          say "The app version '#{version}' is not available as a container (use 'mrsk app containers' for available versions)", :red
+            rolled_back = true
+          else
+            say "The app version '#{version}' is not available as a container (use 'mrsk app containers' for available versions)", :red
+          end
         end
       end
     end
+
+    run_hook "post-rollback", runtime: runtime.round if rolled_back
```

Comment:
I think we can actually just let this be post-deploy as well. Don't think we need a special case for rollback. The outcome is essentially the same: New version is now running.